node/network/collator-protocol/src/validator_side.rs

```diff
@@ -930,7 +930,7 @@ mod tests {
 			overseer_send(
 				&mut virtual_overseer,
 				CollatorProtocolMessage::NetworkBridgeUpdateV1(
-					NetworkBridgeEvent::OurViewChange(View(vec![test_state.relay_parent]))
+					NetworkBridgeEvent::OurViewChange(View::new(vec![test_state.relay_parent]))
```

Comment:
feel like a `view![]` macro wouldreduce the clutter a bit, like in availability-dist